content/modules/redisgears/installing-redisgears.md

```diff
@@ -0,0 +1,144 @@
+---
+Title: Installing RedisGears 
+description:
+weight: 60
+alwaysopen: false
+categories: ["Modules"]
+---
+Before you can use RedisGears, you have to install the RedisGears module on your RS cluster.
+
+## Minimum Requirements
+
+- Redis Enterprise 6.0.0 or later
+- The [cluster is setup]({{< relref "/rs/administering/cluster-operations/new-cluster-setup.md" >}}) and all of the nodes are joined to the cluster
+
+{{< note >}}
```

Comment:
@bmansheim let's remove this note for now.  I might add it in later on but we decided not to support k8s for now